### DISCUSSION TITLE
code review IM-13

### DISCUSSION THREAD

**Discussion by Kirillbiliashov** [HowProgrammingWorks/LiveQA#84] [Code review]
[Курсовая работа](https://github.com/Kirillbiliashov/Coursework), писал на Java. Программа позволяет играть в покер с другими игроками (ботами) в консоли. Автор: Биляшов Кирилл ИМ-13

**Comment by tshemsedinov** [comment]
See review: https://github.com/Kirillbiliashov/Coursework/issues/1
